.github/workflows/ci-rust.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           - os: macos
           - os: ubuntu
           - os: windows
+          - os: windows-11-arm
```

Comment:
This requires https://github.com/actions/partner-runner-images/issues/77 to be merged first, which apparently is currently going through [a last round of tests](https://github.com/actions/partner-runner-images/issues/77#issuecomment-3032556925) and should be published soon 🚀